io.github.lemonade_emu.Lemonade.json

```diff
@@ -0,0 +1,176 @@
+{
+    "app-id": "io.github.lemonade_emu.Lemonade",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.llvm17"
+    ],
+    "command": "lemonade-launcher",
+    "rename-desktop-file": "lemonade-qt.desktop",
+    "rename-mime-file": "lemonade.xml",
+    "rename-icon": "lemonade",
+    "rename-mime-icons": [
+        "lemonade"
+    ],
+    "copy-icon": true,
+    "build-options": {
+        "env": {
+            "CI": "1",
+            "GITHUB_ACTIONS": "1",
+            "GITHUB_REPOSITORY": "Lemonade-emu/Lemonade",
+            "GITHUB_REF_NAME": "master"
+        },
```

Comment:
What is this part for?

Reply:
@bbhtt 
This section specifies environment variables and build options to be used during the build process.

Reply:
I know but why do you need Github CI environment variables here for? The build pipeline does not run on Github CI here and having them in manifest won't affect anything.


Are you using them in the buildsystem then?

Reply:
@bbhtt 
no, this is already from Citra json so we didn't change it btw if it's not required or won't affect anything we can remove this section.